config/confighttp/README.md

```diff
@@ -75,6 +75,8 @@ will not be enabled.
 - [`tls`](../configtls/README.md)
 - [`auth`](../configauth/README.md)
 
+[MemoryLimiterExtension](../../extension/memorylimiterextension/README.md) can be leverage to provide backpressure when the OTEL processor does not have memory resource to accepts anymore data.
```

Comment:
It's unclear that this is another config option
```suggestion
- `memory_limiter`: [Memory limiter extension](../../extension/memorylimiterextension/README.md) that will reject incoming requests once the memory utilization grows above configured limits.
```

Reply:
updated